packages/demand-bloc-payment/package.json

```diff
@@ -32,19 +32,11 @@
   "dependencies": {
     "@adyen/adyen-web": "4.7.4",
     "@karhoo/demand-api": "^16.12.0",
-    "braintree-web": "3.62.2",
-    "core-js-pure": "3.17.2"
-  },
-  "devDependencies": {
-    "@microsoft/api-extractor": "7.7.13",
-    "@types/braintree-web": "3.47.2",
-    "@types/jest": "25.1.3",
-    "ts-jest": "25.2.1",
-    "tsdx": "0.13.1",
-    "typescript": "3.8.3"
+    "braintree-web": "^3.62.2",
+    "core-js-pure": "^3.20.1"
   },
   "peerDependencies": {
     "@adyen/adyen-web": "4.7.4",
-    "braintree-web": "3.62.2"
+    "braintree-web": "^3.62.2"
```

Comment:
These 2 packages (adyen and braintree) are defined in both `dependencies` and `peerDependencies` which is not correct. I think it should be only in peerDeps and also should be added to dev dependencies to work locally